src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/configuration/SubscribeTarget.java

```diff
@@ -16,19 +16,52 @@
 
 package com.exactpro.th2.common.schema.message.impl.rabbitmq.configuration;
 
-import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.ToString;
-
-@Getter
-@Builder
-@ToString
-@EqualsAndHashCode
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
 public class SubscribeTarget {
 
-    private String queue;
+    private final String queue;
+
+    private final String routingKey;
+
+    public SubscribeTarget(String routingKey, String queue) {
+        this.queue = queue;
+        this.routingKey = routingKey;
+    }
+
+    public String getQueue() {
+        return queue;
+    }
+
+    public String getRoutingKey() {
+        return routingKey;
+    }
+
+    @Override
+    public String toString() {
+        return new ToStringBuilder(this)
+                .append("queue", queue)
+                .append("routingKey", routingKey)
+                .toString();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+
+        if (obj == null || getClass() != obj.getClass())
+            return false;
```

Comment:
Please add braces to `if`-statements. Also maybe we should use `instanceof` instead of `getClass() != obj.getClass()`

Reply:
done